### PR TITLE
refactor(SAMS-62): Add aria-labels where possible

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds `aria-label` attributes where ever possible to make it easier for screen readers to access the contents.

It makes use of google chrome developer tool's experimental feature for accessibility to visualize how the content is structured for screen reader:

Screenshot 2022-03-22 at 15 02 38

### CODE REVIEW

**Review by dnsos — COMMENTED** [review]
Great work on this @vogelino!

I fixed a couple of typos and have one point of discussion: Is an `aria-label` necessary when the next element in line actually has text content already? In the best case it's just double-proofed, but a downside could be that there is more maintenance keeping `aria-label`s and texts in sync.

File: src/_includes/partials/ersten_modul.liquid [hunk 1]
```diff
@@ -15,13 +18,14 @@
...
-        Service Design erzeugt gute Angebote, die Stress für Mitarbeiter:innen reduzieren, Geld sparen und Ressourcen freisetzen.
+        Service Design erzeugt gute Angebote, die Stress für Mitarbeiter:innen
+        reduzieren, Geld sparen und Ressourcen freisetzen.
       </p>
     </div>
     <div class="md:relative">
       <img
         src="/assets/images/modules/01_hero.webp"
-        alt="Abstrakte Illustration zum Modul 1: Service Design und Nutzer:innenorientierung"
+        aria-label="Abstrakte Illustration zum Modul 1: Service Design und Nutzer:innenorientierung"
```
[the remark this answers is not in the document]
Comment (vogelino) [reply src/_includes/partials/ersten_modul.liquid:28]: This is what I understood. Aria label will have precedence over the alt attribute.

File: src/_includes/partials/footer.liquid [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
         href="https://citylab-berlin.org"
         class="opacity-100 transition-opacity hover:opacity-80"
         aria-label="Logo vom Projekt CityLab Berlin"
       >
         {% include 'partials/citylab_logo_color.liquid' %}
       </a>
-      <ul class="flex gap-6 sm:gap-16">
-        <li class="flex flex-col gap-4">
+      <ul class="flex gap-6 sm:gap-16" aria-label="Partner Logos">
+        <li class="flex flex-col gap-4" aria-label="Projektautor">
```
[the remark this answers is not in the document]
Comment (vogelino) [reply src/_includes/partials/footer.liquid:21]: As far I understand, users using a screen reader would need to dig deeper in the structure before reading the text content. So it is already accessible, but providing an aria-label higher in the hierarchy means that users can decide earlier to skip to further contents.

**Review by dnsos — APPROVED** [review]
Thanks for the clarifications @vogelino. Let's merge this!

**Pull request merged**